src/apps/clear-all-todo.js

```diff
@@ -0,0 +1,8 @@
+// Clear all completed todos function
+const clearAllCompletedTodos = (localStorageTodos) => {
+  const localArr = JSON.parse(localStorage.getItem(localStorageTodos));
+  const pendingTodos = localArr.filter((list) => list.completed === false);
+  localStorage.setItem(localStorageTodos, JSON.stringify(pendingTodos));
+  window.location.reload();
+};
+export default clearAllCompletedTodos;
```

Comment:
- [ ] great job adding most of the functionalities 👍  however according to the requirements you should Implement a function for deleting a task (remove an element from the array). right now the only way to delete a task is to mark it as complete and then remove all completed items.